api/v1/resourceslice.go

```diff
@@ -41,17 +41,9 @@ type ResourceSliceStatus struct {
 }
 
 type ResourceState struct {
-	// True when the resource has been sync'd to the specified manifest.
```

Comment:
Is this documented anywhere else?

Reply:
These comments are included in the generated CRD yaml files so that's something. But also I think we will need nice hand-written docs later on and they should definitely include this.